docs/data/Allas/using_allas/swift_client.md

```diff
@@ -0,0 +1,380 @@
+
+
+This chapter includes instructions for the following operations:
+
+| Swift command | Function |
+| :---- | :---- |
+| post | Create a bucket |
+| upload | Upload an object |
+| list | List objects and buckets |
+| download | Download objects and buckets |
+| copy | Move object |
+| delete | Remove objects or bucket |
+| download --all | Download whole project |
+| delete --all | Remove whole project |
+| md5sum | Get the checksum |
+| stat | View metadata |
+| stat --meta | Add metadata |
+| tempurl | Create a temporary URL |
+| post -r, -w, --read-acl | Manage access rights |
+| upload --use-slo | Upload files larger than 5 GB |
+
+
+You can install and use Swift also outside CSC computing environment, but in that case make sure that your Swift version is not too old, since the older Swift versions might not work with Allas.
+
+
+## Create buckets and upload objects
+
+Creating a new empty bucket can be done with command:
+```bash
+swift post <new_bucket_name>
+```
+
+You can also create a new bucket and add a file in it with command:
+```bash
+swift upload <new_bucket_name> <file_name>
+```
+
+Adding a file to a existing bucket can be done with the same command:
+```bash
+swift upload <old_bucket_name> <file_name>
+```
+**Note:** This might cause a warning  "_409 Conflict: BucketAlreadyExists_", but that does not necessarily mean that the upload failed. 
+If the next line displays the file name, it was successfully uploaded to the already existing bucket.
+
+```bash
+$ swift upload my_fishbucket my_fish.jpg
+Warning: failed to create container 'my_fishbucket': 409 Conflict: BucketAlreadyExists
+my_fish.jpg
+```
+&nbsp;
+
+
+## List objects and buckets
+
+To list all the buckets belonging to the project use command:
+```bash
+$ swift list
+my_fishbucket
+my_bigfishes
+```
+Listing the content of a certain bucket:
+```bash
+$ swift list my_fishbucket
+my_fish.jpg
+salmon.jpg
+bass.png
+```
+&nbsp;
+
+
+## Download objects and buckets
+
+Downloading an object can be done with command:
+```bash
+swift download <bucket_name> <file_name>
+```
+If you want to rename the object as you download it, you can add <i>-o new_name</i> at the end of the command:
+```bash
+swift download <bucket_name> <file_name> -o <new_name>
+```
+Additionally, you can download a whole bucket at once:
+```bash
+swift download <bucket_name>
+```
+&nbsp;
+
+## Move objects
+
+You can copy data from one bucket to another with command ```swift copy```. The command below copies <i>file.txt</i> from _bucket1_ to _bucket2_.
+```bash
+swift copy --destination /bucket2 bucket1 file.txt
+```
+**Note:** In case there is no bucket called _bucket2_, Swift will create a new bucket with that name. However, even if there is a bucket called _bucket2_, Swift will claim that it created a new one even though it just copied the file to the already existing one:
```

Comment:
There are a lot of these "manual" note things. We should probably use special note box syntax for most of these
!!!note